src/message_replies/formatCode.js

```diff
@@ -0,0 +1,28 @@
+import beautify from 'js-beautify';
+import format from '../utils/jsFormatConfig.js';
+
+const { js: jsBeautify, css: cssBeautify, html: htmlBeautify } = beautify;
+
+const formatCode = async (interaction, prefixCommand) => {
+  if (prefixCommand === '!js') {
+    const stripPrefix = interaction.content.replace(/!js/g, '');
+    const message = jsBeautify(stripPrefix, format);
+    const string = '```js\n' + message + '```';
+    interaction.reply(string);
+  }
+
+  if (prefixCommand === '!css') {
+    const stripPrefix = interaction.content.replace(/!css/g, '');
+    const message = cssBeautify(stripPrefix, format);
+    const string = '```css\n' + message + '```';
+    interaction.reply(string);
+  }
+
+  if (prefixCommand === '!html') {
+    const stripPrefix = interaction.content.replace(/!html/g, '');
+    const message = htmlBeautify(stripPrefix, format);
```

Comment:
those "if's" have the same code, differing only the comparison, if you create a hashtable for the styles, you can easily reuse the code

Reply:
I thought about this, but I think it gets a little murky if we decide to extend it to different languages, so like a python formatter wont use `beautify` thus, the code would no longer be reusable. 

But i do think this could be cleaner with more reusable code, e.g. the stripping of the prefix can be a helper and the creation of the message. 

Will refactor.